project/src/main/webapp/events-feed.js

```diff
@@ -11,5 +15,48 @@ async function getAllEvents() {
 function populateAllEvents(allEvents) {
   for (const event of allEvents) {
     populateEventContainer(event, 'event-container');
+    getInterestFilters(event);
+  }
+  populateFilters(filters)
+}
+
+const filters = {}
```

Comment:
This might just be personal preference, but if you declare globals, I'd group them at the top of the file (or under the document.ready function) so they don't get lost in the middle of code and decoupled from the method they're supposed to be associated with over time

Reply:
done